airflow/executors/celery_executor.py

```diff
@@ -212,6 +211,8 @@ def trigger_tasks(self, open_slots: int) -> None:
             num_processes = min(len(task_tuples_to_send), self._sync_parallelism)
 
             send_pool = Pool(processes=num_processes)
+            import pydevd_pycharm
+            pydevd_pycharm.settrace('10.0.9.198', port=56374, stdoutToServer=True, stderrToServer=True)
```

Comment:
Should we create a pre-commit hook to check for dev-tools/commands in the code?

Reply:
Yeah! For sure. Will do.

Reply:
Done in #6662 @feluelle !